src/services/search/dealerSearch.ts

```diff
@@ -0,0 +1,40 @@
+import { postData, Service } from "../../base"
+
+import { Paginated } from "../../types/pagination"
+import { DealerParams } from "../../types/params/dealer"
+
+import { SearchDealer } from "../../types/models/dealer"
+
+const defaultPagination = {
+  page: 0,
+  size: 3,
+}
+
+const searchForDealers = (path, searchQuery: DealerParams) => {
+  const { pagination, sort, query } = searchQuery
+  const { page, size } = pagination
+
+  const sizeOrDefault =
```

Comment:
This looks like the same code as in https://github.com/carforyou/carforyou-api-client-pkg/blob/0b273d6a63ecda60d07899ea1e83d3d3b993375d/src/services/search/listingSearch.ts#L55 can you move it to a  library to avoid duplication, thanks 🙂

Reply:
oki doki

Reply:
done